app/views/orga/teams/show.html.slim

```diff
@@ -82,6 +84,17 @@ nav.btn-group
       = links_to_conferences([preference.conference]).first
       = preference_option(preference)
 
+- if can? :see_offered_conferences, @team
+
+  - if @team.conference_attendances.present?
+    h4 Conference offers
+    dl
+      - @team.conference_attendances.each do |offer|
+        dt Offered conference:
+        dd = offer&.conference&.name
+        - if offer.orga_comment.present?
+          dt Orga comment:
+          dd = offer&.orga_comment
```

Comment:
No need for the lonely operator here, you're already checking for the presence 2 lines above.